libarchive/archive_write_set_format_pax.c

```diff
@@ -66,6 +66,7 @@ struct pax {
 	unsigned flags;
 #define WRITE_SCHILY_XATTR       (1 << 0)
 #define WRITE_LIBARCHIVE_XATTR   (1 << 1)
+#define WRITE_GENERIC_KW	 (1 << 2)
```

Comment:
Indentation is off if you care.  (need another space between W and the left paren)

Reply:
Will fix, thanks.